test/issue-104.test.js

```diff
@@ -0,0 +1,57 @@
+const t = require('tap')
+const test = t.test
+const FindMyWay = require('../')
+
+test('Parametric route, url with parameter common prefix > 1', t => {
```

Comment:
Can you also add a test with the same code of the issue? :)

Reply:
Added, test called `Nested static parametric route, url with parameter common prefix > 1` :)